ConfigSpace/io/pcs_new.py

```diff
@@ -0,0 +1,475 @@
+                  "float": UniformFloatHyperparameter,
+                  "categorical": CategoricalHyperparameter,
+                  "ordinal": OrdinalHyperparameter}
+
+        try:
+            param_list = pp_cont_param.parseString(line)
+            log = param_list[10:]
+            if len(log) > 0:
+                log = log[0]
+            param_list = param_list[:10]
+            name = param_list[0]
+            paramtype = "int" if "integer" in param_list else "float"
+            lower = float(param_list[3])
+            upper = float(param_list[5])
+            log_on = True if "log" in log else False
+            default = float(param_list[8])
+            param = create[paramtype](name=name, lower=lower, upper=upper,
+                                      q=None, log=log_on, default=default)
+            cont_ct += 1
+        except pyparsing.ParseException:
+            pass
+
+        try:
+            if "categorical" in line:
+                param_list = pp_cat_param.parseString(line)
+                name = param_list[0]
+                choices = [choice for choice in param_list[3:-4:2]]
+                default = param_list[-2]
+                param = create["categorical"](name=name, choices=choices, default=default)
+                cat_ct += 1
+                
+            elif "ordinal" in line:
+                param_list = pp_ord_param.parseString(line)
+                name = param_list[0]
+                sequence = [seq for seq in param_list[3:-4:2]]
+                default = param_list[-2]
+                param = create["ordinal"](name=name, sequence=sequence, default=default)
+                ord_ct += 1
+                
+        except pyparsing.ParseException:
+            pass
+
+        if param is None:
+            raise NotImplementedError("Could not parse: %s" % line)
+
+        configuration_space.add_hyperparameter(param)
+
+    for clause in forbidden:
+        param_list = pp_forbidden_clause.parseString(clause)
+        tmp_list = []
+        clause_list = []
+        for value in param_list[1:]:
+            if len(tmp_list) < 3:
+                tmp_list.append(value)
+            else:
+                # So far, only equals is supported by SMAC
+                legal = ['=', '>', '<']
+                if tmp_list[1] in legal:
+                    # TODO maybe add a check if the hyperparameter is
+                    # actually in the configuration space
+                    clause_list.append(ForbiddenEqualsClause(
+                        configuration_space.get_hyperparameter(tmp_list[0]),
+                        tmp_list[2]))
+                else:
+                    raise NotImplementedError()
+                tmp_list = []
+        configuration_space.add_forbidden_clause(ForbiddenAndConjunction(
+            *clause_list))
+
+
+    conditions_per_child = []
+    for condition in conditions:
+        conditions_per_child.append(condition)
+    for condition in conditions_per_child:
```

Comment:
If I read the code below correctly, it only support two conditions. Could you please extend the code such that it supports an arbitrary amount of conditions?